docker/django/Dockerfile

```diff
@@ -91,14 +95,18 @@ CMD python manage.py rqworker --with-scheduler
 FROM python-base as web-dev
 
 CMD python /opt/bigcases2/manage.py migrate  && \
+    python /opt/bigcases2/manage.py tailwind install --no-input  && \
```

Comment:
Why do these need to get installed every time? Aren't they part of our code base?

Reply:
This is the command that runs the `npm install` and creates a `node_modules` folder with all the dependencies. We're ignoring this folder so We need to create it before We use the `build` command